pkgs/by-name/ve/veloren/package.nix

```diff
@@ -0,0 +1,125 @@
+{ lib
+, rustPlatform
+, fetchFromGitLab
+, runCommand
+, pkg-config
+, vulkan-loader
+, alsa-lib
+, udev
+, shaderc
+, xorg
+, libxkbcommon
+}:
+
+let
+  # Note: use this to get the release metadata
+  # https://gitlab.com/api/v4/projects/10174980/repository/tags/v{version}
+  version = "0.15.0";
+  date = "2023-06-30-15:09";
+  rev = "75c3b8ca659b6b740ebea8db46d2ebccb5bac6c6";
+
+  src = fetchFromGitLab {
+    owner = "veloren";
+    repo = "veloren";
+    inherit rev;
+    hash = "sha256-TKs/WT5S1hUBd9Y7AivFwbhlMI92GA5Auaevrjnx1cE=";
+  };
+
+  # Game assets, should not to be cached by hydra
+  assets = lib.dontDistribute (runCommand "veloren-assets" {} ''
```

Comment:
Note: I'm [told](https://matrix.to/#/%23dev%3Anixos.org/%24zw9una3U1xAgRtK-kmsdhMKb4deACquUjELl2nxV0qY?via=maxwell.ydns.eu&via=matrix.org&via=snug.si&via=turbo.ooo) this is probably not going to work.
Anyway, the assets are just 321MB, which I suppose is acceptable for the cache.